hack/manifest-gen/manifest-gen.sh

```diff
@@ -73,9 +73,10 @@ while getopts "cug" OPT; do
             shift $((OPTIND-1))
             (
                 cd "$SCRIPT_DIR"
-                go build -o manifest-gen >/dev/null 2>&1
-                ./manifest-gen --source="$EFFECTIVE_SRC_CHART_DIR" generate "$@"
-                rm manifest-gen
+                tmpBinary=$(mktemp /tmp/manifest-gen.XXXXXX)
```

Comment:
What is the motivation for this change?

Reply:
It is to be able to introduce parallelism. It's due to a few things:
- This script is called by the `generate` make target
- We run the CI container by mounting the current directory `-v $(ROOT_DIR):$(GO_MOUNT_PATH)`, so several runs of the CI container in parallel share the same directory
- several steps depend on the `generate` target
So, when we run in parallel  `make -C .ci TARGET=xxx` and `make -C .ci TARGET=yyy ci`, there may be a race condition where the binary generated by `xxx` is removed by `yyy`.

At one point I was massively testing parallelism and got the problem. It may not be really necessary now but I left it.